scheduler/test/cook/test/mesos/api.clj

```diff
@@ -1216,4 +1220,43 @@
                (api/retrieve-url-path target-framework-id agent-hostname "executor-102")))
         (is (nil? (api/retrieve-url-path target-framework-id agent-hostname "executor-103")))
         (is (= (str "http://" agent-hostname ":5051/files/read.json?path=%2Fpath%2Ffor%2Fexecutor-111")
-               (api/retrieve-url-path target-framework-id agent-hostname "executor-111")))))))
\ No newline at end of file
+               (api/retrieve-url-path target-framework-id agent-hostname "executor-111")))))))
+
+(deftest test-instance-progress
```

Comment:
Can we then just write to the db directly instead of involving the `handle-status-update`?